src/ert/shared/hook_implementations/workflows/disable_parameters.py

```diff
@@ -3,18 +3,57 @@
 
 class DisableParametersUpdate(ErtScript):
     """The DISABLE_PARAMETERS workflow disables parameters,
-    so they are excluded from the update step. The job takes a list
+    so they are excluded from the update step. The job takes a group
     of parameters as input:
 
-    DISABLE_PARAMETERS "PARAMETER_1, PARAMETER_2"
+    DISABLE_PARAMETERS PRED
 
     The parameters that are given as arguments will be removed from the
-    update. Note that if giving more than one parameter as input the list
-    must be enclosed in quotes.
+    update.
 
     Note that unknown parameter names will be silently ignored.
 
-    This workflow is recommended to be run as a PRE_FIRST_UPDATE hook
+    This workflow is recommended to be run as a PRE_FIRST_UPDATE hook.
+
+    As example to disable parameters in the poly.ert case (located in
+    ert/test-data/poly_example) has three steps:
+
+    First, in the ert config file ("poly.ert"), include:
+
+        LOAD_WORKFLOW disable_parameter_pred
+        HOOK_WORKFLOW disable_parameter_pred PRE_FIRST_UPDATE
+
+        GEN_KW PRED coeff.tmpl coeffs_pred.json coeff_priors
+        GEN_KW COEFFS coeff.tmpl coeffs.json coeff_priors
+
+    Note that the output file from the groups should be different to avoid overwriting
+    values sampled.
+
+    Second, make a new file called "disable_parameter_pred" with:
+
+        DISABLE_PARAMETERS PRED
+
+    Finally, analyse the plots. There would be two groups of parameters:
+
+        COEFFS:COEFF_A
+        COEFFS:COEFF_B
+        COEFFS:COEFF_C
+
+        PRED:COEFF_A
+        PRED:COEFF_B
+        PRED:COEFF_C
+
+    All the parameters in the COEFFS group are updated though the iterations, while the
+    PRED has the same sampling through the iterations.
+
+    Historical note:
+    GEN_KW PRED used to hold a special meaning and be excluded from being updated.
+    The special meaning was replaced by the DisableParametersUpdate workflow though the
+    DISABLE_PARAMETERS key instead.
+    The objective to use it is to only update some parameters in the AHM process,
+    but let them vary when running the prediction after the AHM.
+    So the disabled parameters won't be updated during AHM but that will be used in
+    prediction.
```

Comment:
Think this should either be a comment in the code, or it can be read through the git history.

Reply:
ok, the I will change the name of "GEN_KW PRED" to "GEN_KW DISABLE_PRED". Otherwise the user still gets the warning related to "GEN_KW PRED". Not documenting the historical reasons to get the warning seems confusing to me, so renaming it seems more reasonable.